.gitignore

```diff
@@ -0,0 +1,50 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
you can't add this in this project (you must set this configuration in your global gitignore file)